app/scripts/controllers/product/CreateChargeController.js

```diff
@@ -116,12 +116,11 @@
                             else{
                                 scope.showenablefreewithdrawal = false;
                                 scope.showenablepaymenttype = false;
-                            }
-                            }
-                            else {
                                 scope.showdatefield = false;
                                 scope.repeatEvery = false;
                             }
+                            }
+
```

Comment:
datefield and repeatEvery part was not visible in the suitable conditions, this is fix for it